react/components/form/BaseInput.js

```diff
@@ -39,17 +39,17 @@ const BaseInput = props => {
       error={!!currentError}
       errorMessage={currentError}
       helpText={schema.description}
-      label={<FormattedMessage id={label}/>}
-      max={max}
-      min={min}
+      label={<FormattedMessage id={label} />}
```

Comment:
This shouldn't be passing the `<FormattedMessage />` component, just the `label`